### DISCUSSION TITLE
Better handling of Mill plugins

### DISCUSSION THREAD

**Discussion by ckipp01** [scalacenter/scaladex#1063] [Ideas]
In more recent versions of Mill there is the convention of publishing Mill plugins in a similar way that Scala JS or Scala Native publish. For example in your mill build you might see:

```scala
import $ivy.`com.goyeau::mill-scalafix::0.2.10`
```

However, if you're using Mill 0.10.x the actual artifact it fetches is `mill-scalafix_mill0.10_2.13`. So when you're including a badge for this artifact like you see in https://index.scala-lang.org/joan38/mill-scalafix/badges you see that entire name. I think it'd be nice to special case mill plugins and show that really this is just `mill-scalafix` for `Mill (0.10.x)`. This would be similar to how you differentiate between JVM and JS artifacts.

**Comment by adpi2** [comment]
Thanks for opening this discussion! This makes total sense. Mill should be a platform like sbt is a platform. It would make badges of Mill plugin more readable and also it would allow people to browse all Mill plugins on a particular Mill version.

**Reply by adpi2** [reply]
@ckipp01 I am working on it: https://index-dev.scala-lang.org/search?q=*&platforms=mill0.10

**Reply by ckipp01** [reply]
Ahh amazing!

**Reply by ckipp01** [reply]
~There is actually a whole handful of plugins I don't see on this page. For example none of:~

- https://github.com/ckipp01/mill-giter8
- https://github.com/ckipp01/mill-ci-release
- https://github.com/ckipp01/mill-github-dependency-graph
- https://github.com/ckipp01/mill-scip

~Do I need to add something in my projects to get them to appear here?~

Ahh, nevermind, I realize this is `index-dev`

**Reply by adpi2** [reply]
Here they are: https://index.scala-lang.org/search?platforms=mill0.10

**Reply by ckipp01** [reply]
Very cool, great job on this @adpi2!
